NOTICE

```diff
@@ -0,0 +1,9 @@
+Apache NiFi - MiNiFi
+Copyright 2014-2016 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+This product includes the following work from the Apache Hadoop project:
+
+BoundedByteArrayOutputStream.java adapted to SoftLimitBoundedByteArrayOutputStream.java
```

Comment:
Should this notice information be in the root level pom of MiNiFi or as part of the Notice when NiFi dependencies are brought in?